pressbooks/pressbooks
# Replace print pdf icon

This PR fixes https://github.com/pressbooks/pressbooks/issues/2264 and https://github.com/pressbooks/pressbooks/issues/2266

To test:
1. Produce a Print PDF export
2. Check to see that the new icon appears on the export page. The icon should have a red banner and should include the text 'Print PDF'

**Note:** I did not replace the 36 and 72 px versions of this image as they are not used anywhere in Pressbooks. The large size is explicitly specified for the export table, which is the only place these icons are currently used in Pressbooks: https://github.com/pressbooks/pressbooks/blob/86b921b5c3ede76e3fd94732e56cce50c710ded8/inc/modules/export/class-table.php#L414-L416
